src/oss/langchain-runtime.mdx

```diff
@@ -8,8 +8,134 @@ import AlphaCallout from '/snippets/alpha-lc-callout.mdx';
 
 ## Overview
 
+LangChain's `create_agent` runs on LangGraph's runtime under the hood.
+LangGraph exposes a `Runtime` object with the following information:
+
+1. Context: static information like user id, db connections, or other dependencies for an agent invocation
+2. Store: a `BaseStore` instance used for long term memory
+3. Stream writer: an object used for streaming information via the `"custom""` stream mode
+
 ## Access
 
+When creating an agent with `create_agent`, you can specify a `context_schema` to define
+the structure of the `context` stored in the agent runtime.
+
+You can access the runtime information within [tools](#inside-tools), [prompt](#inside-prompt), and [pre and post model hooks](#inside-pre-and-post-model-hooks).
+
+When invoking the agent, pass the `context` argument with the relevant configuration for the run:
```

Comment:
nit: this sentence talks about "passing" by the previous one about "accessing" -- feels like a gap in terms of presentation